### DISCUSSION TITLE
Some questions about the military helicopter

### DISCUSSION THREAD

**Discussion by boltyboi** [TEHE-Studios/ExpandedHelicopterEvents#306] [Q&A] [answered]
I've already read in a previous thread that it'll spare anybody wearing 'police' or 'military' tagged clothes:

![image](https://github.com/TEHE-Studios/ExpandedHelicopterEvents/assets/58641698/8a6523b0-9649-445a-9c57-defbe4691821)

I'm just wondering if this refers to the name of the item itself or something that the dev who made said clothes has manually tagged them as.

As an example, will any clothes exclusively spawning in places like the louisville military barracks/ army surplus stores (or the clothes on any zombie soldiers) protect me from the military helicopter **regardless of the mod**?

Also, is there a minimum amount of these clothes I need to be wearing, e.g. can I just wear an army jacket and jeans for it to 'recognise me as a soldier'? Do i need to wear full fatigues?

Thanks in advance

**Comment by Chuckleberry-Finn** [comment, marked as answer]
The code is found here:
https://github.com/TEHE-Studios/ExpandedHelicopterEvents/blob/52aa45e431236df2d080b40ae9d0524e5f907628/Contents/mods/Expanded%20Helicopter%20Events/media/lua/shared/ExpandedHelicopter02a_Presets.lua#L104-L127

But basically, it uses the `type` of the item (looking for Police, Army, Military, or Riot) and/or if the item is specifically tagged (this has to be done by the other modder). The count to determine if someone looks too 'civilian' is 3 pieces of clothing.

**Reply by boltyboi** [reply]
good stuff man, thanks for the info
